### PR TITLE
73745a

### PR DESCRIPTION
**Pull request by sophy-colbert**

### CODE REVIEW

File: app/views/recommendations/73745a.html.erb [hunk 1]
```diff
@@ -10,15 +10,92 @@
...
     <div class="location-details">
-      <%= map(latitude: 53.833567, longitude: -1.503924) %>
+      <%= map(latitude: 53.833567, longitude: -1.503927) %>
       <dl>
         <dd>35 minutes walk</dd>
         <dd class="availability">Next session: Tuesday 16 January 9.30am</dd>
       </dl>
     </div>
   </section>
+  <h1><span>Other people near you want to sing.</span> You're not alone!</h1>
```
Comment (jamesarice) [inline app/views/recommendations/73745a.html.erb]: This line doesn't seem like it belongs here.

File: app/views/recommendations/73745a.html.erb [hunk 2]
```diff
@@ -10,15 +10,92 @@
...
+  <div class="other-people">
+    <img src="<%= image_path 'anonymous.svg' %>" alt="">
+  </div>
+  <section class="start-something-detail">
+    <h2>Start something new</h2>
```
Comment (jamesarice) [inline app/views/recommendations/73745a.html.erb]: `Start something new` isn't really what they're doing here.

File: app/views/recommendations/73745a.html.erb [hunk 3]
```diff
@@ -10,15 +10,92 @@
...
+    <p>We’re recommending this activity to one other person in your neighbourhood. Would you like me to introduce you to them?</p>
+    <%= interest_button "I'm interested in this" %>
```
Comment (jamesarice) [inline app/views/recommendations/73745a.html.erb]: Suggest changing the button text to something like `Yes, introduce me`

File: app/views/recommendations/73745a.html.erb [hunk 4]
```diff
@@ -10,15 +10,92 @@
...
         <dd>£7 per session</dd>
-        <dd>Sign up here: <a href="https://ukoutdoorfitness.com/venue/roundhay-park/">https://ukoutdoorfitness.com/venue/roundhay-park/</a></dd>
+        <dd>Sign up here: <a href="https://ukoutdoorfitness.com/venue/roundhay-park/" target="_blank">https://ukoutdoorfitness.com/venue/roundhay-park/</a></dd>
       </dl>
     </header>
     <div class="location-details">
-      <%= map(latitude: 53.833567, longitude: -1.503924) %>
+      <%= map(latitude: 53.833567, longitude: -1.503927) %>
       <dl>
         <dd>35 minutes walk</dd>
```
Comment (jamesarice) [inline app/views/recommendations/73745a.html.erb]: Might it be useful to show the address of the place?
As well as `Get directions` after `35 minute walk` making sure not to include user's postcode in the Google Maps link, you can create a directions map with a blank starting point to the destination. eg. `https://www.google.co.uk/maps/dir//Roundhay+Park,+Mansion+Ln,+Leeds+LS8+2HH/@53.82939,-1.5759173,12z/data=!3m1!4b1!4m9!4m8!1m0!1m5!1m1!1s0x48795b0b0acc019f:0x58de428758fe0d6e!2m2!1d-1.5058778!2d53.8294104!3e2`

File: app/views/recommendations/73745a.html.erb [hunk 5]
```diff
@@ -10,15 +10,92 @@
...
+    <h2>Start something new</h2>
+    <p>We’re recommending this activity to one other person in your neighbourhood. Would you like me to introduce you to them?</p>
+    <%= interest_button "I'm interested in this" %>
+    <p class="helper-text">(Only your first name and email address will be shared)</p>
+  </section>
+
+<% end %>
+
+<%= recommendation id: 'road-closure', why: 'You said you’d like to organise to close the street for playing' do %>
+  <h1><span>Fancy closing Barncroft Drive for a play day?</span> A road closure doesn’t usually cost anything, however you will have to show that you’ve got agreement from all your neighbours who will be affected. Leeds City Council can help you get started.</h1>
```
Comment (jamesarice) [inline app/views/recommendations/73745a.html.erb]: The street name is too identifiable in combination with their name. Can we generalise this?

File: app/views/recommendations/73745a.html.erb [hunk 6]
```diff
@@ -10,15 +10,92 @@
...
+        <dd>30 minutes by bus and walking</dd>
+        <dd class="availability">Next class: Saturday 20 January</dd>
+      </dl>
+    </div>
+  </section>
+<% end %>
+
+
+<%= recommendation id: 'leeds-repair-cafe', why: 'You said you’d like to learn to fix my broken phone' do %>
+  <h1><span>Don’t replace - repair!</span> Get your broken stuff mended and learn how to repair things yourself at a Repair Cafe. You can join the Facebook group to find out when the next cafe is, or get help starting your own event. They happen all over Leeds so you may have to travel - I’ve attended one of these events and they are really fun, and you’d be amazed what can be saved from the dump.</h1>
```
Comment (jamesarice) [inline app/views/recommendations/73745a.html.erb]: General comment on all these conversational intros. There's quite a lot of words on the card, multiplied by half a dozen cards it's quite a lot to read. Can they be more concise?
screen shot 2018-01-10 at 15 25 17

File: app/views/recommendations/73745a.html.erb [hunk 7]
```diff
@@ -10,15 +10,92 @@
...
+    <header>
+      <dl>
+        <dt>LS14 Trust</dt>
+        <dd>See Facebook page for next event</dd>
+        <dd>No cost</dd>
+        <dd>Talk to someone who’s done it before: info@ls14trust.org</dd>
+      </dl>
+    </header>
+    <div class="location-details">
+      <%= map(latitude: 53.827285, longitude: -1.469097) %>
```
Comment (jamesarice) [inline app/views/recommendations/73745a.html.erb]: There's a map, but no details below it. Can we add these? Eg. Address, time to get there etc.

File: app/views/recommendations/73745a.html.erb (hunk 4 above)
Comment (sophy-colbert) [reply app/views/recommendations/73745a.html.erb]: I don't think we can do this if we're keeping personal stuff out of git.

Comment (sophy-colbert) [reply app/views/recommendations/73745a.html.erb]: I've added in some details about the meeting place.

File: app/views/recommendations/73745a.html.erb (hunk 2 above)
Comment (sophy-colbert) [reply app/views/recommendations/73745a.html.erb]: Done

File: app/views/recommendations/73745a.html.erb (hunk 3 above)
Comment (sophy-colbert) [reply app/views/recommendations/73745a.html.erb]: Done

File: app/views/recommendations/73745a.html.erb (hunk 5 above)
Comment (sophy-colbert) [reply app/views/recommendations/73745a.html.erb]: Done

File: app/views/recommendations/73745a.html.erb (hunk 7 above)
Comment (sophy-colbert) [reply app/views/recommendations/73745a.html.erb]: added address

**Pull request merged**